Makefile

```diff
@@ -26,11 +26,16 @@ all_generated_files := \
 .DEFAULT_GOAL := help
 
 ##@ Dependencies
+.PHONY: install-build-deps
+install-all-deps: install-build-deps install-site-deps ## Install all dependencies for building both binary and user docs)
 
 .PHONY: install-build-deps
 install-build-deps: ## Install dependencies (packages and tools)
 	./install-build-deps.sh
-	pip3 install -r userdocs/requirements.txt
+
+.PHONY: install-site-deps
+install-site-deps: ## Install dependencies for user docs
+	pip install -r userdocs/requirements.txt
```

Comment:
Hum. At least in the case of Ubuntu, the tool is called `pip3` - not sure about other distributions or MacOS.

Reply:
good point, `pip3` is safer.

Reply:
Good point, I have changed to pip3 to be safe.